### PR TITLE
Add Illes Balears

### PR DESCRIPTION
**Pull request by gonzaloserrano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,5 @@
 - https://festivus-es.github.io/festivus/Espa%C3%B1a/Comunidad%20de%20Madrid/Madrid/festivus.ics
 - https://festivus-es.github.io/festivus/Espa%C3%B1a/Comunitat%20Valenciana/Paterna/festivus.ics
 - https://festivus-es.github.io/festivus/Espa%C3%B1a/Comunitat%20Valenciana/Burjassot/festivus.ics
+- https://festivus-es.github.io/festivus/Espa%C3%B1a/Illes%20Balears/Palma%20de%20Mallorca/festivus.ics
+- https://festivus-es.github.io/festivus/Espa%C3%B1a/Illes%20Balears/Sóller/festivus.ics
```
Comment (gonzaloserrano) [inline README.md:10]: hmm not sure if need to urlencode this

**Comment by alexpdp7** [comment]
Thanks! Merged! I'll wait until CI runs and see if it needs tweaking, but looks good!

**Comment by gonzaloserrano** [comment]
I checked and both calendars were imported correctly 👍

**Comment by alexpdp7** [comment]
Yes, I checked the URLs, but not the ICS themselves.

Thanks for your PR!

**Pull request merged**